main.py

```diff
@@ -441,13 +441,13 @@ def show_sidebar():
     ROOT.minsize(180, 90)
 
     # Setting the title of the window.
-    ROOT.title("D-Pad")
+    ROOT.title(" D-Pad")
```

Comment:
Why is there blank space before the title? 

> `ROOT.title('_D-Pad')`

Reply:
I thought the window title looks congested.

Reply:
> I thought the window title looks congested.
Okay, in that case, I will let it be there.